models/export.py

```diff
@@ -197,20 +197,33 @@ def load_state_dict(model, state_dict, train, exclude_anchors):
 
             print(f'{prefix} starting export with onnx {onnx.__version__}...')
             f = opt.weights.replace('.pt', '.onnx')  # filename
-            if not sparseml_wrapper.enabled:
-                torch.onnx.export(model, img, f, verbose=False, opset_version=opt.opset_version, input_names=['images'],
-                                  dynamic_axes={'images': {0: 'batch', 2: 'height', 3: 'width'},  # size(1,3,640,640)
-                                                'output': {0: 'batch', 2: 'y', 3: 'x'}} if opt.dynamic else None)
-            else:
-                # export through SparseML so quantized and pruned graphs can be corrected
-                save_dir = '/'.join(f.split('/')[:-1])
-                save_name = f.split('/')[-1]
-                exporter = ModuleExporter(model, save_dir)
-                exporter.export_onnx(img, name=save_name, convert_qat=True)
-                try:
-                    skip_onnx_input_quantize(f, f)
-                except:
-                    pass
+            # export through SparseML so quantized and pruned graphs can be corrected
+            save_dir = '/'.join(f.split('/')[:-1])
```

Comment:
Any particular reason we are using string join instead of os.path.join?

Reply:
This is what the original code was written as, I just unindented it. I'll update to use the os libraries, though